src/rsa/pss_verify_tests.txt

```diff
@@ -0,0 +1,60 @@
+# Test vectors for RSA-PSS padding for variable public modulus lengths
```

Comment:
I'll move these test vectors into the same file as the NIST test vectors. In the PKCS#1 test vectors and other places we already mix imported test vectors with others that we've created, so there's always one file per algorithm (basically). It makes the regeneration of the converted-from-NIST files less convenient but it's not worth having separate files.

Plus, this file doesn't have the right naming convention.